config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Dependency.java

```diff
@@ -8,45 +8,52 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 import java.util.Objects;
 
-import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY;
-
-@JsonInclude(NON_EMPTY)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@JsonPropertyOrder({"type", "name"})
 public class Dependency {
-    private  String id;
     @JsonIgnore
-    private  Class<? extends GatewayEntity> type;
-    private  String name;
-    private  String entityType;
-    public Dependency(){
+    private String id;
+    @JsonIgnore
+    private Class<? extends GatewayEntity> typeClass;
```

Comment:
take the latest changes from feature branch these names are changed

Reply:
@uppoju which branch?

Reply:
base feature branch